bundles/edu.kit.kastel.sdq.eclipse.grading.view/META-INF/MANIFEST.MF

```diff
@@ -15,3 +15,7 @@ Bundle-RequiredExecutionEnvironment: JavaSE-17
 Bundle-ActivationPolicy: lazy
 Bundle-Activator: edu.kit.kastel.eclipse.grading.view.activator.Activator
 Automatic-Module-Name: edu.kit.kastel.sdq.eclipse.grading.view
+Import-Package: org.eclipse.jface.text,
+ org.eclipse.jface.text.source,
+ org.eclipse.ui.editors.text,
+ org.eclipse.ui.texteditor
```

Comment:
Typically, we use require bundle if possible. I'll have a look at it.

Reply:
Eclipse did this automatically when importing the respective packages

Reply:
You have two options: "Add Required Bundle"  or add "Import Package" (both should be options; however, we normally use Require Bundle if possible, because the dependencies are explicit) You can try to remove the statements and use the auto-complete option of eclipse (same menu as you've already used :) )

Reply:
Using require-bundle doesn't work for me with org.eclipse.jface.text.source (which contains the ISourceViewer). It works for all the other packages though. Sadly, I have no idea how Eclipse plugins and OSGi work, so I'm not sure why it works with import-package but not with require-bundle